plan4past/utils/ppnf_visitor.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2021 -- 2023 WhiteMech
+#
+# ------------------------------
+#
+# This file is part of Plan4Past.
+#
+# Plan4Past is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Plan4Past is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Plan4Past.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Pnf visitor."""
```

Comment:
```suggestion
"""PPNF visitor."""
```

Reply:
why two "P"? I guess you meant to use just one?

Reply:
Fixed in [edc66cc](https://github.com/whitemech/Plan4Past/commit/edc66ccc624a97d70f69bb9ee5d008bd240b6de3)

Reply:
If I'm not wrong, this should be the Propositional PNF (PPNF). That's why two "P"s

Reply:
fixed